pkg/util/timeUtil.go

```diff
@@ -0,0 +1,19 @@
+package util
+
+import (
+	"github.com/golang/protobuf/ptypes/timestamp"
+	"time"
+)
+
+// Default layout of string representation.
+const defautlLayout = time.RFC3339
+
+// ToTime returns a time.Time from a *timestamp.Timestamp.
+func ToTime(t *timestamp.Timestamp) time.Time {
```

Comment:
These utils are in the ptypes package

https://github.com/golang/protobuf/blob/master/ptypes/timestamp.go